draft-ietf-quic-http.md

```diff
@@ -727,12 +727,14 @@ indicating different stream IDs, but MUST NOT increase the value they send in
 the last Stream ID, since clients might already have retried unprocessed
 requests on another connection.  A server that is attempting to gracefully shut
 down a connection SHOULD send an initial GOAWAY frame with the last Stream ID
-set to the maximum value allowed by QUIC's MAX_STREAMS and SHOULD NOT increase
-the MAX_STREAMS limit thereafter.  This signals to the client that a shutdown is
+set to the maximum value allowed by the concurrency control of QUIC for the
```

Comment:
Is this 2^62-4 or the current maximum stream limit (for c-i, bidi)?

Concretely, I think that if we are making a recommendation, we should permit either.  The difference between those choices should be clear enough to implementers once we lay it out like that.

Reply:
Yes. This PR is (or have been) about agreeing both. I've added "currently"  at 1d54261, and made editorial adjustments in c4a513c.

Reply:
FWIW, it is my understanding that the last non-editorial change regarding GOAWAY was #1712, which had "currently". It seems that we have dropped that while making editorial changes.